### ISSUE TITLE
support labels option for StratifiedSplitter

### ISSUE THREAD

**Issue by corochann** [pfnet-research/chainer-chemistry#206]
support to receive `labels` or maybe `y`? so that we can use it with numpy array, similar way to the sklearn.
http://scikit-learn.org/stable/modules/generated/sklearn.model_selection.StratifiedKFold.html
